applications/registration-react/src/config.js

```diff
@@ -70,5 +70,7 @@ export const loadConfig = async () => {
   //   SSO_HOST: 'http://localhost:8084'
   // });
 
+  // use local server on host.
+  env.REGISTRATION_API_HOST = 'http://localhost:8115';
```

Comment:
This is probably not meant to be committed

Reply:
fikset